views/maintainer_group.py

```diff
@@ -25,3 +27,5 @@ def get_object(self, *args, **kwargs):
             return MaintainerGroup.objects.get(pk=1)
         except IndexError:
             raise Http404
+        except ObjectDoesNotExist:
```

Comment:
Use `MainatainerGroup.DoesNotExists`. And remember to delete the import statement.